src/program/protocols/union/accept_pegin.rs

```diff
@@ -509,12 +509,7 @@ impl AcceptPegInProtocol {
             reveal_tx_name,
             OutputSpec::Index(won_enabler.1 as usize),
             operator_won_tx_name,
-            InputSpec::Auto(
-                SighashType::taproot_all(),
-                SpendMode::KeyOnly {
-                    key_path_sign: SignMode::Aggregate,
-                },
-            ),
+            InputSpec::Auto(SighashType::taproot_all(), SpendMode::Script { leaf: 0 }),
```

Comment:
I think we must start declaring the leaf indexes as constants with proper names to make it easier to reason (and update) which leaves are we referring to.